test/teos/unit/test_api.py

```diff
@@ -28,6 +27,8 @@
     ENCRYPTED_BLOB_MAX_SIZE_HEX,
 )
 
+INTERNAL_API_ENDPOINT = "localhost:50051"
```

Comment:
We are already importing config from `test.teos.conftest`, so you can get the data from there instead of declaring again.

Reply:
Done in 3f3a29f.